algoliasearch-common/src/main/java/com/algolia/search/responses/DeleteUserID.java

```diff
@@ -0,0 +1,24 @@
+package com.algolia.search.responses;
+
+import java.io.Serializable;
+
+public class DeleteUserID implements Serializable {
+
+  private String deletedAt;
+
+  @SuppressWarnings("unused")
+  public String getDeletedAt() {
+    return deletedAt;
+  }
+
+  @SuppressWarnings("unused")
+  public DeleteUserID setDeletedAt(String deletedAt) {
+    this.deletedAt = deletedAt;
+    return this;
+  }
+
+  @Override
+  public String toString() {
+    return "DeleteUserID{" + "deletedAt='" + deletedAt + '\'' + '}';
```

Comment:
It's a nit but maybe using String.Format would be more clearer ?

Reply:
Agree but as it seems to be written like this everywhere else in the code, I think @julienbourdeau followed the same pattern for consistency which I'd agree with.

Reply:
Yes, you're right, we can leave it like that for consistency. :)